src/main.rs

```diff
@@ -629,10 +681,27 @@ compatibility with the original codemod.",
                 .long("extensions")
                 .value_name("EXTENSION")
                 .multiple(true)
-                .require_delimiter(true)
```

Comment:
need to leave this delimiter for codemod interoperability, think you just removed the wrong one